extensions/blocks/calendly/icon.js

```diff
@@ -1,26 +1,14 @@
 /**
  * External dependencies
  */
-import { SVG, G, Path, Rect } from '@wordpress/components';
+import { SVG, Path } from '@wordpress/components';
 
 export default (
 	<SVG height="24" viewBox="0 0 23 24" width="23" xmlns="http://www.w3.org/2000/svg">
-		<G fill="none" fillRule="evenodd">
-			<Rect
-				height="20.956522"
-				rx="3"
-				stroke="#656a74"
-				strokeWidth="2"
-				width="20.956522"
-				x="1"
-				y="2.043478"
-			/>
-			<Rect fill="#656a74" height="4.869565" rx="1" width="2" x="6.565217" />
-			<Rect fill="#656a74" height="4.869565" rx="1" width="2" x="14.652174" />
-			<Path
-				d="m14.6086957 10.0869565c-.6956522-.57971012-1.6231885-.8695652-2.7826087-.8695652-1.7391305 0-3.47826091 1.5652174-3.47826091 3.6521739s1.73913041 3.6521739 3.47826091 3.6521739c1.1594202 0 2.0869565-.3478261 2.7826087-1.0434782"
-				stroke="#656a74"
-			/>
-		</G>
+		<Path d="M19,1h-2.3v0c0-0.6-0.4-1-1-1c-0.6,0-1,0.4-1,1v0H8.6v0c0-0.6-0.4-1-1-1c-0.6,0-1,0.4-1,1v0H4C1.8,1,0,2.8,0,5
+	v15c0,2.2,1.8,4,4,4h15c2.2,0,4-1.8,4-4V5C23,2.8,21.2,1,19,1z M21,20c0,1.1-0.9,2-2,2H4c-1.1,0-2-0.9-2-2V5c0-1.1,0.9-2,2-2h2.6
+	v0.8c0,0.6,0.4,1,1,1c0.6,0,1-0.4,1-1V3h6.1v0.8c0,0.6,0.4,1,1,1c0.6,0,1-0.4,1-1V3H19c1.1,0,2,0.9,2,2V20z M13.9,14.8l1.4,1.4
+	c-0.9,0.9-2.1,1.3-3.5,1.3c-2.4,0-4.5-2.1-4.5-4.7s2.1-4.7,4.5-4.7c1.4,0,2.5,0.4,3.4,1.1L14,10.9c-0.5-0.4-1.2-0.6-2.1-0.6
+	c-1.2,0-2.5,1.1-2.5,2.7c0,1.6,1.3,2.7,2.5,2.7C12.7,15.5,13.4,15.3,13.9,14.8z"/>
```

Comment:
I am an SVG amateur. Could you explain what's changed here?

Reply:
I imported the SVG code into Illustrator and cleaned it up a bunch. Essentially where we were using shapes (rectangles, paths) I combined everything into a single path that can be more easily addressable. In the process, I also removed the inline color definitions.